internal/playwright/config.go

```diff
@@ -75,8 +76,15 @@ func FromFile(cfgPath string) (Project, error) {
 	}
 
 	// Default project path
-	if p.Playwright.ProjectPath == "" {
-		return Project{}, fmt.Errorf("no project folder defined")
+	if p.Playwright.ProjectPath == "" && p.RootDir == "" {
+		return Project{}, fmt.Errorf("could not find 'playwright.projectPath' or 'rootDir' in config yml, one of those two must be set to a directory containing your project")
+	}
+
+	if p.Playwright.ProjectPath != "" && p.RootDir != "" {
+		log.Info().Msgf(
+			"Found both 'playwright.projectPath=%s' and 'rootDir=%s' in config. Defaulting to rootDir '%s'",
```

Comment:
Is this useful ? `projectPath` should disappear, so maybe warning is only useful when there is only `projectPath` set ?
If both are set, I'm not sure we should write out anything about this.

Reply:
@FriggaHel it's just to warn them that they're using both and that projectPath is being ignored. Users may wonder why "projectPath" isn't working.

Reply:
@dpgraham in that case, I would add one message: 
1. warn of deprecation of `projectPath`
2. warn of deprecation & ineffectiveness of `projectPath` (if `rootDir` is set in config)